### PR TITLE
vte: configurable color palette

### PR DESCRIPTION
**Pull request by ii8**
#7

### CODE REVIEW

File: src/tsm/tsm-vte.c [hunk 1]
```diff
@@ -514,32 +339,22 @@ void tsm_vte_unref(struct tsm_vte *vte)
 }
 
 SHL_EXPORT
-void tsm_vte_set_osc_cb(struct tsm_vte *vte, tsm_vte_osc_cb osc_cb, void *osc_data)
+void tsm_vte_set_osc_cb(struct tsm_vte *vte, tsm_vte_osc_cb osc_cb,
+			void *osc_data)
```
Comment (Aetf) [inline src/tsm/tsm-vte.c]: Unrelated change

**Comment by ii8** [comment]
Ok, I might have another look at this if I get around to it at some point.

**Comment by Aetf** [comment]
Thank you anyway! I'll take a look at this since I get some spare time now.
